video_xblock/static/html/youtube.html

```diff
@@ -2,10 +2,9 @@
   id="{{ video_player_id }}"
   class="video-js vjs-default-skin"
   controls
-  height="560"
+  height="607"
```

Comment:
I believe Youtube worked fine before this change.

Reply:
I faced same issue on Youtube too while testing it.